packages/node-cli/package.json

```diff
@@ -15,12 +15,8 @@
 		"url": "https://github.com/oracle/netsuite-suitecloud-sdk/issues"
 	},
 	"repository": "https://github.com/oracle/netsuite-suitecloud-sdk/tree/master/packages/node-cli",
-	"main": "./src/suitecloud.js",
 	"preferGlobal": true,
 	"sdkDownloadUrl": "https://system.netsuite.com/download/suitecloud-sdk/cli/java/latest",
-	"bin": {
```

Comment:
I think you can keep the bin defintion and the cli file and just add `index.js`, no? not an js expert but I think we should do as less changes as possible to be able to benfit from future relases. If possible we should submit the PR to orinigal repo.